lib/validators/commit.js

```diff
@@ -17,6 +17,12 @@ class Commit extends Validator {
       'pull_request_review.*'
     ]
     this.supportedSettings = {
+      jira: {
+        enabled: 'boolean',
```

Comment:
We should remove all references to `enabled` since we've agreed it's not relevant.